target/linux/generic/config-6.12

```diff
@@ -117,6 +117,7 @@ CONFIG_32BIT=y
 # CONFIG_ADIS16475 is not set
 # CONFIG_ADIS16480 is not set
 # CONFIG_ADI_AXI_ADC is not set
+# CONFIG_ADI_AXI_DAC is not set
```

Comment:
> generic: 6.12: add more missing symbols

Squash it and add a `Co-authored-by:` tag.

Reply:
Done.